src/task/task.ts

```diff
@@ -101,12 +103,27 @@ export default class TaskImpl<T> implements Task<T> {
     );
   }
 
-  async enable(): Promise<void> {
+  async enable(options?: TaskEnablingOptions): Promise<void> {
+    let patchOp = async () => this._data.patch(() => ({ enabled: true }));
+
+    // Patch task with enabling options
+    if (options) {
+      if (!!options.skipPastOccurrences && this._data.nextRunTime) {
+        const nextRunTime = computeNextRun(
+          this._data.interval,
+          this._data.nextRunTime.getTime(),
```

Comment:
wouldn't you want this to be based on the current time? otherwise, this could end up pushing your next run way out. if for instance your schedule was once per week and your next run is scheduled in 6 days, this would push it out to 13 days

Reply:
sure makes sense.

Reply:
In that case I wouldnt have to change the code in computeNextRun and it will just base off of the current time. Ill revert those changes as well